internal/workflow/processing_test.go

```diff
@@ -571,6 +666,10 @@ func (s *ProcessingWorkflowTestSuite) TestAMWorkflow() {
 		A3m:          a3m.Config{ShareDir: s.CreateTransferDir()},
 		Preservation: pres.Config{TaskQueue: temporal.AmWorkerTaskQueue},
 		Storage:      storage.Config{DefaultPermanentLocationID: amssLocationID},
+		ValidatePREMIS: premis.Config{
+			Enabled: true,
+			XSDPath: "/home/enduro/premis.xsd",
+		},
```

Comment:
... and here. That way it will be tested in the a3m and the AM branches from the workflow. All other test cases could remain as they are.

Keep the preservation action/task IDs changes in all of them, those are great. But remove the config and the activity call from all except these two.